specs/monitoring_integration_service.adoc

```diff
@@ -0,0 +1,207 @@
+= Spec - Monitoring-integration as a service
+
+== Introduction
+
+Monitoring Integration component is responsible for the integration of tendrl with grafana.
+It performs the following functionalities:
+
+* Uploads default dashboards to grafana.
+
+== Problem description
+
+In the present implementation of monitoring-integration to upload dashboards in
+grafana the user has to run the code manually every time which is not feasible.
+Monitoring integration will be providing other functionalities such as
+
+* Handling alerts generated by grafana
+* Making dynamic configuration changes in graphite and grafana
+
+for which it will be required to be implemented as a service.
+
+== Proposed change
+
+Monitoring-integration component is changed to a service based on the tendrl structure.
+Instead of starting each and every times manually, it is installed and started once
+as a service in the background.
+
+To convert as a service monitoring-integration component needs some additional
```

Comment:
i will change like "To be able to run as a daemon, the component needs some additional changes"